openshift-sdn.spec

```diff
@@ -0,0 +1,141 @@
+#debuginfo not supported with Go
+%global debug_package %{nil}
+%global gopath      %{_datadir}/gocode
+%global osdn_gopath _output/local/go
+%global gopkg_base  github.com/openshift
+%global import_path %{gopkg_base}/openshift-sdn
+%global commit      1d03a6c75a44a638b9f2590b1475a6c980ce829d
+
+Name:           openshift-sdn
+Version:        0.4
+Release:        1%{?dist}
+Summary:        SDN solutions for OpenShift
+License:        ASL 2.0
+URL:            https://%{import_path}
+ExclusiveArch:  x86_64
+Source0:        https://%{import_path}/archive/%{commit}/%{name}-%{version}.tar.gz
+
+BuildRequires:  systemd
+BuildRequires:  golang >= 1.2-7
+
+Requires:       openvswitch >= 2.3.1
+Requires:       bridge-utils
+
+%description
+%{summary}
+
+%package master
+Summary:        Openshift SDN Master
+Requires:       openshift-sdn = %{version}-%{release}
+Requires:       openshift-master
+
```

Comment:
Need these Requires for master and node
~~~
Requires(post): systemd
Requires(preun): systemd
Requires(postun): systemd
~~~
http://fedoraproject.org/wiki/Packaging:ScriptletSnippets#New_Packages